zenodo/modules/records/data/objecttypes.json

```
@@ -712,4 +713,24 @@
     "type": "publication"
   },
   "csl": "article"
+},
+{
+  "$schema": "https://zenodo.org/schemas/records/objecttype-v1.0.0.json",
+  "internal_id": "publication-peerreview",
+  "id": "https://zenodo.org/objecttypes/publication/peerreview#",
+  "title": {
+    "en": "Peer review"
+  },
+  "title_plural": {
+    "en": "Peer reviews"
+  },
+  "schema.org": "https://schema.org/ScholarlyArticle",
+  "datacite": {"general": "Text", "type": "Peer review"},
```

```suggestion
  "datacite": {"general": "PeerReview"},
```

This one's important so that we index on DataCite the right general type (see https://schema.datacite.org/meta/kernel-4.4/doc/DataCite-MetadataKernel_v4.4.pdf)